src/web/src/app/app-routing.module.ts

```diff
@@ -9,17 +9,24 @@ import { spaceGuard } from './space/guard';
 import { LogoutComponent } from "./auth/logout/logout.component";
 import { LoginComponent } from "./auth/login/login.component";
 import { UserTestComponent } from "./user/user/user.component";
-import { HelloComponent } from "./space/hello/component";
+import { SpaceListComponent } from "./space/list/component";
+import { SpaceAddComponent } from "./space/add/component";
+import { SpaceDetailComponent } from "./space/detail/component";
+import { SpaceRequestSentComponent } from './space/request-sent/component';
 import { TgAuthComponent } from './tg-auth/tg-auth.component';
 
 const routes: Routes = [
 	{path: 'login', component: LoginComponent, data: {title: 'NAVIGATION.LOGIN'}},
+	{path: 'login/complete', component: LoginComponent, data: {title: 'NAVIGATION.LOGIN', complete: true}},
 	{path: 'logout', component: LogoutComponent, data: {title: 'NAVIGATION.LOGOUT'}},
-	{path: 'hello', component: HelloComponent, data: {title: 'NAVIGATION.HELLO'}, canActivate: [authGuard]},
+	{path: 'logout/complete', component: LogoutComponent, data: {title: 'NAVIGATION.LOGOUT', complete: true}},
 	{path: '', canActivateChild: [authGuard, spaceGuard], children: [
 		{path: '', redirectTo: '/dashboard', pathMatch: 'full'},
 		{path: 'usertest', component: UserTestComponent, data: {title: 'NAVIGATION.LOGOUT'}},
-
+		{path: 'space/list', component: SpaceListComponent, data: {title: 'NAVIGATION.SPACELIST'}},
+		{path: 'space/add', component: SpaceAddComponent, data: {title: 'NAVIGATION.SPACEADD'}},
+		{path: 'space/:id', component: SpaceDetailComponent, data: {title: 'NAVIGATION.SPACEDETAIL'}},
```

Comment:
Извини, пропустил сразу этот момент. Тут должен быть key а не id. Т.е. везде в ссылках которые видит пользователь используем key, а при обращении к бэку используем id
```suggestion
		{path: 'space/:key', component: SpaceDetailComponent, data: {title: 'NAVIGATION.SPACEDETAIL'}},
```

Reply:
Не проблема. Готово.